cmd/btfs/functest.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	testfile        = "QmZHeNJTU4jFzgBAouHSqbT2tyYJxgk6i15e7x5pudBune"
+	testfile        = "QmPdsdr4JznLs2FFFhjDbMzrnXmNfny8wpKXETKfbXdeK7"
```

Comment:
Why we need to update this value? Is there hash function change?

Reply:
> Why we need to update this value? Is there hash function change?
yes, metadata has been added to the file